Issue 9 Contact Us Page

tylerz1443 opened the pull request.

sanehirakenji reviewed (changes requested):
Hey Tyler, similar to my comments for the previous pr, is this how you wanted it formatted? 
<img width="741" alt="Contactus" src="https://github.com/volunthree/volunthree/assets/97561440/0ec859a5-a3c0-46a6-950e-b8fef14ffb5c">
It seems like you meant for those to be in different rows.

Please also update the comment and the routing to this page.

tylerz1443 commented:
ok think I fixed everything and I have updated the issue to the current version of main.  I have not added the page to the app.jsx page yet, but if both of these pages are fine I will added it to one of these pages for both of them and the push it as an update.
